CMakeLists.txt

```diff
@@ -20,20 +21,10 @@ message(STATUS "Debug mode ON" )
 else(DEBUG STREQUAL "ON")
 # Release mode
 set(CMAKE_BUILD_TYPE Release)
-set(CMAKE_C_FLAGS "-O4")
 message(STATUS "Debug mode OFF")
 
 endif (DEBUG STREQUAL "ON")
 
-set(LIBRARY_OUTPUT_PATH ${PROJECT_SOURCE_DIR}/bin/${CMAKE_BUILD_TYPE})
```

Comment:
Changed later to use cmake builddir, i.e. to obey distro builddir (set by e.g. cmake --build builddir).